src/pages/ProductPage/ProductPage.jsx

```diff
@@ -1,14 +1,21 @@
 import { Card } from "../../component/Card/Card";
+import { ProductsFilter } from "../../component/ProductsFilter/ProductsFilter";
 import { useCard } from "../../component/Context/CardContext/CardContext";
```

Comment:
suggestion: add an index.js file in the component folder and export all components from index.js.

Reply:
Thanks for pointing it out